GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -278,7 +279,7 @@ private void LayoutRevisionInfo()
             if (AppSettings.ShowRevisionInfoNextToRevisionGrid.ValueOrDefault)
             {
                 RevisionInfo.Parent = RevisionsSplitContainer.Panel2;
-                RevisionsSplitContainer.SplitterDistance = RevisionsSplitContainer.Width - 500;
+                RevisionsSplitContainer.SplitterDistance = RevisionsSplitContainer.Width - 420;
```

Comment:
@KindDragon I suppose that at this point controls are not scaled yet. Can you check it under debugger?